src/clj/ataru/applications/application_store.clj

```diff
@@ -206,6 +207,7 @@
   [application applied-hakukohteet old-answers form update? connection]
   (let [flat-form-content   (-> form :content util/flatten-form-fields)
         fields-by-id        (util/form-fields-by-id form)
+        excluded-attachment-ids-when-yo-and-jyemp (util/attachment-ids-from-children form higher-base-education-module-id)
```

Comment:
Pitäis käsittääkseni välittää `(:content form)` ensimmäisenä parametrina.

Reply:
OK logiikka lisätty